queries.sql

```diff
@@ -8,3 +8,70 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg<10.5;
 SELECT * FROM animals WHERE neutered=TRUE;
 SELECT * FROM animals WHERE name!='Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Delete all animals born after Jan 1st, 2022 */
+BEGIN;
+DELETE FROM animals
+WHERE  date_of_birth > '2022-01-01';
+SELECT * FROM animals;
+
+/* Create a savepoint */
+SAVEPOINT my_savepoint;
+
+/* Update all animals' weight to be their weight multiplied by -1 */
+UPDATE animals
+SET weight_kg = weight_kg * -1;
```

Comment:
- [ ] You only added one transaction. You should add 3 other transactions according to the requirements in the screenshot below. Please add those transactions.

![image](https://github.com/eamador1/vet-database/assets/68796232/736245c3-cef6-4951-a030-e644485dca37)